Removing sexually related objects

KeplerWasTaken opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What does it change? What other things could this impact? -->

Removes a lot of sex pins and some sort of maid-catgirl thing

**Media**
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged.

Only put changes that are visible and important to the player on the changelog.

Don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers

Putting a name after the :cl: symbol will change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB
-->

:cl:
- remove: Sexual stuff; this is not an ERP server

KeplerWasTaken committed:
Removing sexually related objects

KeplerWasTaken committed:
Removes more sexual stuff

KeplerWasTaken committed:
Removed sexual images

MasterSneed commented:
We have already added middle eastern localization to hippie

Chudjak commented:
Some of these I agree with, some of these I do not.
Nyanify is the same as catbeasting, maid is funny and no cumbrains will dissuade me of that and cat ears should remain in the code simply due to the fact that they always were there. We aren't an ERP server, if anyone uses these to ERP they just get banned.
Keep it to only the pins.

Acensti committed:
update

cursed item should stay in game, super syndicate glue it and then give to someoen

Chudjak commented:
Not as a syndicate item.
Add the stuff I told you back.

KeplerWasTaken commented:
okay
I will add in the smites back

Chudjak commented on Resources/Prototypes/Catalog/uplink_catalog.yml:
```diff
@@ -845,6 +845,19 @@
+  description: uplink-cat-ears-desc
+  productEntity: ClothingHeadHatCatEarsCursed
+  cost:
+    Telecrystal: 26
```
wtf?

KeplerWasTaken committed:
Readded catgirl stuff because you wanted it

KeplerWasTaken committed:
Merge branch 'removefluff' of https://github.com/KeplerWasTaken/hippies-station-14 into removefluff

KeplerWasTaken committed:
Delete ghost of uplink cat ears

KeplerWasTaken commented:
okay the cat shit has been re-added, still have yet to test

Chudjak commented:
it's been a week, you tested it yet?

KeplerWasTaken commented:
ye ill test it now. testing ss13 crap always been a pain, even worse with the ugly ass UI of ss14. Dark mode is retarded

KeplerWasTaken commented:
has some bugs working on it now.

KeplerWasTaken committed:
Made sure it works now

KeplerWasTaken commented:
okay bugs gone game launches bye sex

BarginMart commented:
Good Riddance
